ChangeLog

```diff
@@ -1,6 +1,28 @@
 # ChangeLog for Pod::Simple dist
 #---------------------------------------------------------------------------
-3.44 2022-xx-yy Karl Williamson <khw@cpan.org>
+3.47 2025-05-xx Karl Williamson <khw@cpan.org>
```

Comment:
@khwilliamson, I don't quite understand your use of `xx-yy` here.  When do these variables get filled in with actual values?

Reply:
Thanks; fixed